core/connection/session.go

```diff
@@ -25,14 +25,43 @@ import (
 	"github.com/mysteriumnetwork/node/session"
 )
 
+// SessionStatus represents list of possible session statuses
+type SessionStatus int
```

Comment:
We can just use `iota` here to enumerate statuses.

Reply:
is it good idea to use iota with statuses stored in database?